third-party/realdds/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
```

Comment:
Dont we want here the minimal cmake version for DDS capability?

Reply:
Kind of optional -- we have minimums all over the place, mostly needlessly.
The user will get stuck on the FastDDS CMake before they get here.